### PR TITLE
OCPBUGS-14462: update channel name in persistent-storage-local-install.adoc

### PR DESCRIPTION
**Pull request by dobsonj**
Version(s):

4.11 and newer

Issue:

https://issues.redhat.com/browse/OCPBUGS-14462

Link to docs preview:

https://60824--docspreview.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent_storage_local/persistent-storage-local.html#local-storage-install_persistent-storage-local

QE review:
- [ ] QE has approved this change.

Additional information:

/label peer-review-needed
/cc @openshift/storage @openshift/storage-qe @lpettyjo

### CODE REVIEW

**Comment by GroceryBoyJr** [comment]
Hello @dobsonj : Please, can you add this link to your comment 0: https://60824--docspreview.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent_storage_local/persistent-storage-local.html#local-storage-install_persistent-storage-local
This preview link will help guide reviewers to what your changes look like when rendered into the docs.

**Comment by GroceryBoyJr** [comment]
Also: @dobsonj , lpettyjo is off duty right now. I'll do a peer review but if you are able to wait for her to get back, I agree with you that she should see it also, since this falls into her documentation specialization.

**Comment by dobsonj** [comment]
> Also: @dobsonj , lpettyjo is off duty right now. I'll do a peer review but if you are able to wait for her to get back, I agree with you that she should see it also, since this falls into her documentation specialization.

@GroceryBoyJr ack, thanks, appreciate the review! I think this PR can wait until Lisa and QE have a chance to review as well.

**Comment by GroceryBoyJr** [comment]
@dobsonj A good choice sir. I expect she will recommend adding a callout (a #2 item explaining the yaml change) for the "stable" channel change. I'll provide an example but I encourage having her review that also.

File: modules/persistent-storage-local-install.adoc [hunk 1]
```diff
@@ -98,7 +89,7 @@ metadata:
   name: local-storage-operator
   namespace: openshift-local-storage
 spec:
-  channel: "${OC_VERSION}"
+  channel: stable
```
Comment (GroceryBoyJr) [inline modules/persistent-storage-local-install.adoc:92]:
```suggestion
  channel: stable <2>
```

Comment (GroceryBoyJr) [reply modules/persistent-storage-local-install.adoc:92]: The companion 2nd item would go below the <1> item and read something like:
<2> The stable channel is required for access to the needed operator.

Comment (GroceryBoyJr) [reply modules/persistent-storage-local-install.adoc:92]: Please feel free to edit or change my wording on a technical basis. I'm only alerting you to point out that the new "stable" channel might need to be identified so the customer is more likely to see it.

**Review by GroceryBoyJr — COMMENTED** [review]
Only two small nits. Fine work sir. Please stand by for review later by lpettyjo.

**Comment by dobsonj** [comment]
@GroceryBoyJr Thanks a lot for the review, I made the change you suggested.
@lpettyjo Please take a look when you have a moment.

File: modules/persistent-storage-local-install.adoc (hunk 1 above)
Comment (chao007) [reply modules/persistent-storage-local-install.adoc]: Thanks.

**Comment by dobsonj** [comment]
/close
this was fixed in https://github.com/openshift/openshift-docs/pull/62715

**Pull request closed without merging**